core/src/test/scala/better/files/FilesSpec.scala

```diff
@@ -399,6 +399,8 @@ class FilesSpec extends FlatSpec with BeforeAndAfterEach with Matchers {
     }
     /***************************************************************************/
     import java.nio.file.{StandardWatchEventKinds => Events}
+    import FileWatcher._
+
     import akka.actor.{ActorRef, ActorSystem}
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/pathikrit/better-files/file/1107185645/issues/source?bid=2860326&fileBranchId=2894564#l404)